cmake/os.cmake

```diff
@@ -26,9 +26,6 @@ endif()
 add_definitions(
 	"-DQT_USE_FAST_CONCATENATION"
 	"-DQT_USE_FAST_OPERATOR_PLUS"
-	# TODO: Uncomment the following definitions when the resulting errors are fixed.
-	#"-DQT_NO_CAST_FROM_ASCII"
-	#"-DQT_NO_CAST_TO_ASCII"
 )
```

Comment:
This block was moved here by mistake, it should probably be in the root `CMakeLists.txt`.

Reply:
These options have been set by now anyways. No need to move it around now :point_up: